### PR TITLE
refactor(patterns/headings): add padding bottom

### PR DESCRIPTION
**Pull request by caqueste**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: src/docs/Foundations/Typography/HeadingStyles/index.mdx [hunk 1]
```diff
@@ -108,6 +108,12 @@ The color of the underline allows you to give visual context to your page or sec
 
 <Preview path="typographyHeading-underlines" nude />
 
+#### Bottom padding
+
+Headings provide with a default bottom padding to keep a right rythm between the other elements of the interface.
```
Comment (nherwegh) [inline src/docs/Foundations/Typography/HeadingStyles/index.mdx:113]: #### Padding bottom 

Headings include a default padding bottom to maintain layout consistency across breakpoints.

or something like that?

Comment (caqueste) [reply src/docs/Foundations/Typography/HeadingStyles/index.mdx:113]: Here it's a deal about the vertical rhythm inside a page, not about the breakpoints.

Comment (nherwegh) [reply src/docs/Foundations/Typography/HeadingStyles/index.mdx:113]: Headings include a default padding bottom to keep the right vertical rhythm between the interface elements.

then ;-)

**Review by tiloyi — APPROVED** [review]

**Review by nherwegh — APPROVED** [review]

**Review by mohamedMok — APPROVED** [review]

**Comment by nherwegh** [comment]
@caqueste didn't we put a `56px` padding bottom on `XXL` cases?

**Pull request merged**